internal/service/wafv2/schemas.go

```diff
@@ -1079,10 +1079,11 @@ func actionToUseSchema() *schema.Schema {
 		MaxItems: 1,
 		Elem: &schema.Resource{
 			Schema: map[string]*schema.Schema{
```

Comment:
Have you considered refactoring here?

Seems like the challenge feature has been added throughout a number of PRs:
https://github.com/hashicorp/terraform-provider-aws/commit/6774a21cb557057e1c5813f9e8faab692edb3800
https://github.com/hashicorp/terraform-provider-aws/commit/13228fd9ce67ee91c04ee8b4dc6a5301d170db08

The same map schema is maintained at 3 places, so it seems inevitable that this issue will happen again. 

As the documentation references back the `action_to_use` to the `action` block, it would make sense to create an action schema and use that.